quantum/quantum.h

```diff
@@ -149,28 +149,26 @@ extern layer_state_t default_layer_state;
 
 //Function substitutions to ease GPIO manipulation
 #ifdef __AVR__
-    #define PIN_ADDRESS(p, offset) _SFR_IO8(ADDRESS_BASE + (p >> PORT_SHIFTER) + offset)
-
     #define pin_t uint8_t
-    #define setPinInput(pin) PIN_ADDRESS(pin, 1) &= ~ _BV(pin & 0xF)
-    #define setPinInputHigh(pin) ({\
-            PIN_ADDRESS(pin, 1) &= ~ _BV(pin & 0xF);\
-            PIN_ADDRESS(pin, 2) |=   _BV(pin & 0xF);\
-            })
+    #define setPinInput(pin) DDRx_ADDRESS(pin) &= ~ _BV(pin & 0xF)
```

Comment:
@jackhumbert ?

Reply:
In this case, I like using the traditional 'do {} while (0)` idiom instead of the gcc extension.